backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -48,6 +50,7 @@ import java.util.Locale
 data class CalendarParams(
     val selectionMode: SelectionMode,
     val range: ClosedRange<LocalDate> = LocalDate.now()..LocalDate.now().plusYears(1),
+    val selectableMonthRange: ClosedRange<YearMonth> = range.start.yearMonth()..range.endInclusive.yearMonth(),
```

Comment:
I think this should be a parameter of CalendarSelectionMode.SelectWholeMonth instead

Reply:
Good idea, I've updated